src/web/nginx.conf

```diff
@@ -54,31 +54,38 @@ http {
     ssl_protocols       TLSv1.2 TLSv1.3;
     ssl_prefer_server_ciphers   on;
     ssl_ciphers "EECDH+ECDSA+AESGCM EECDH+aRSA+AESGCM EECDH+ECDSA+SHA384 EECDH+ECDSA+SHA256 EECDH+aRSA+SHA384 EECDH+aRSA+SHA256 EECDH+aRSA+RC4 EECDH EDH+aRSA HIGH !RC4 !aNULL !eNULL !LOW !3DES !MD5 !EXP !PSK !SRP !DSS";
-    # disabling unwanted http methods
+    
+    # Disabling unwanted http methods
     if ($request_method !~ ^(DELETE|GET|POST)$ ){
       return 405;
     }
-    # prevent clickjack attacks
-    add_header X-Frame-Options "SAMEORIGIN";
-    # prevent XSS attacks
-    add_header X-XSS-Protection "1; mode=block";
-    # Configuring HSTS
-    add_header Strict-Transport-Security 'max-age=31536000; includeSubDomains; preload';
+
+    # If user agent is a program/bot, block
+    if ($http_user_agent ~* "java|curl|python") {
+      return 403;
+    }
+    
+    # Headers
+    add_header X-Frame-Options "SAMEORIGIN";                                                      # prevent clickjack attacks
+    add_header X-XSS-Protection "1; mode=block";                                                  # prevent XSS attacks
+    add_header Strict-Transport-Security 'max-age=31536000; includeSubDomains; preload' always;   # Configuring HSTS
+    add_header CC-X-Request-ID $request_id;
+    add_header X-Cache_Status $upstream_cache_status;
+
     # Secure Diffie-Hellman for TLS
     ssl_dhparam /etc/nginx/cert/${HOST}.pem;
 
     ssl_session_cache shared:TLS:10m;
     ssl_session_timeout 10m;
-    ssl_buffer_size 4k;
-
-    # Set HSTS to 365 days
-    add_header Strict-Transport-Security 'max-age=31536000; includeSubDomains; preload' always;
+    
+    # Enable session tickets
+    ssl_session_tickets on;
 
     # OCSP stapling
     ssl_stapling on;
     ssl_stapling_verify on;
-    resolver 1.1.1.1 1.0.0.1 [2606:4700:4700::1111] [2606:4700:4700::1001]; # Cloudflare
-
+    resolver 8.8.8.8 8.8.4.4 208.67.222.222 208.67.220.220 valid=60s;
```

Comment:
What is this pointing to?

Reply:
classic Google and Cisco... should have a comment on that!